app/views/companies/edit.html.erb

```diff
@@ -1,4 +1,15 @@
 <div class="container-fluid">
+ <% if @company.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@company.errors.count, "erro", "erros") %> encontrados:</h2>
```

Comment:
Essa mensagem parece ter sido utilizada em mais de uma view, o que acha de extrair para o I18n?

Reply:
Inclusive se for utilizar sempre a mesma estrutura daria pra extrair tudo pra uma partial na verdade.

Reply:
Fizemos um commit alterando a mensagem de erro dos forms para ser um partial na pasta shared.